.github/workflows/build-ci.yml

```
@@ -0,0 +1,24 @@
+name: PMIx Standard CI
+
+on:
+  pull_request:
+    # We don't need this to be run on all types of PR behavior
+    # See
+    #  - https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request
+    #  - https://frontside.com/blog/2020-05-26-github-actions-pull_request
+    types:
+      - opened
+      - synchronize
+      - edited
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Check out the code
+        uses: actions/checkout@v1
+      # Run the container tester
+      - name: Build and Test
+        run: docker run --rm -v ${GITHUB_WORKSPACE}:/home/pmixer/pmix-standard jjhursey/pmix-standard /bin/bash -c "cp -R /home/pmixer/pmix-standard pmix-build && cd pmix-build && make"
```

I went ahead and created the `pmix` organization on docker hub (and added you @jjhursey as an owner) if we eventually want to switch to that top-level namespace:

```suggestion
        run: docker run --rm -v ${GITHUB_WORKSPACE}:/home/pmixer/pmix-standard pmix/pmix-standard /bin/bash -c "cp -R /home/pmixer/pmix-standard pmix-build && cd pmix-build && make"
```

---

Thanks for setting that up. I was thinking of moving those github repos into the pmix or openpmix GitHub namespace anyway, so it would make sense to rename the container namespace at the same time. I'll put that on my near-ish term todo list. We can make this change once that is moved over.